skbio/tree/_tree.py

```diff
@@ -3269,3 +3269,42 @@ def unpack_by_func(self, func):
                 nodes_to_unpack.append(node)
         for node in nodes_to_unpack:
             node.unpack()
+
+    @experimental(as_of="0.5.5-dev")
+    def to_networkx(self):
+        import networkx as nx
```

Comment:
Should be up top :)

Reply:
Agreed. The only thing is that if we want optional imports, we'd need to have a try/except at the top - which means that flake8 need to modified to ignore this error.

Reply:
I don't think we want to introduce optional imports unless there is already precedence within scikit-bio

Reply:
...rather, I think this would motivate including it as a dependency

Reply:
The main reason why I brought this up was previous discussions regarding having matplotlib as an optional dependency, since it having more required dependencies can lead to more maintenance burden.  I've seen other packages use optional dependencies to help mitigate these issues (i.e. pandas with the xlrd dependency for reading excel notebook).

That being said, I don't have super strong feelings about this, so I'll include networkx as a required dependency until someone objects.

Reply:
@ebolyen @ElDeveloper @thermokarst any comment on preference for networkx being an optional or required dependency? It looks like it's already installed with QIIME 2 2020.2, so I don't think it would increase burden more than any other dependency 

@mortonjt I don't think the dependency question should hold up the rest of the implementation IMO.